api/room/models.py

```diff
@@ -36,12 +37,19 @@ class Room(Base, Utility):
         ForeignKey('locations.id', ondelete="CASCADE"),
         nullable=True
     )
+    structure_id = Column(
+        Integer,
+        ForeignKey('structure.id', ondelete="CASCADE"),
+        nullable=True   # TODO: Change this to False
+                        # after migrating from v1 to v2
+    )
```

Comment:
To clarify, is this to prevent a migration error due to data already present in the tags table?

Reply:
No in the `room` table itself. There is already existing data in the room table.

Reply:
Okay.